contained-innersource-enables-collaborative-product-development.md

```diff
@@ -0,0 +1,35 @@
+## Title  
+Contained inner source enables collaborative product development
+
+## Context
+* Multiple business units need a software component.
+* It makes sense for the business units to collaborate; each needs some different changes to the code.
+* One BU doesn't have the resources to do the development for all the rest.
+* InnerSource programs and tools exist.
+
+## Problem  
+The company needs to develop a software product component; multiple business units use this component.
```

Comment:
We can move this to context. So would the Problem be:
The uncontained inner sourcing of the collaboration is regarded as distracting from the needed efficiency of specific collaborative development with the other BUs sharing the use of this component, but normal development practices do not effectively work with sharing of development resources to work on this component.